addrmgr/addrmanager_test.go

```diff
@@ -46,7 +46,7 @@ func TestAddOrUpdateAddress(t *testing.T) {
 		t.Fatal("address manager should contain no addresses")
 	}
 
-	// Attempt to add a non-routable address.
+	// Attempt to add an unroutable address.
```

Comment:
It doesn't really matter since I've seen both used, but non-routable is technically the more correct term.

Reply:
Changed back to non-routable.